presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -46,6 +46,7 @@ statement
     | INSERT INTO qualifiedName columnAliases? query                   #insertInto
     | DELETE FROM qualifiedName (WHERE booleanExpression)?             #delete
     | ALTER TABLE from=qualifiedName RENAME TO to=qualifiedName        #renameTable
+    | COMMENT ON TABLE qualifiedName IS string                         #commentTable
```

Comment:
Is this ANSI SQL compliant syntax? Should it be something which begins from `ALTER`?